addons/account/models/account_report.py

```diff
@@ -483,12 +483,12 @@ def create(self, vals_list):
         return result
 
     def write(self, vals):
-        if 'formula' not in vals:
+        if 'formula' not in vals and vals.get('engine') != 'tax_tags':
             return super().write(vals)
 
         self._strip_formula(vals)
 
-        tax_tags_expressions = self.filtered(lambda x: x.engine == 'tax_tags')
+        tax_tags_expressions = self.filtered(lambda x: x.engine == 'tax_tags' or vals.get('engine') == 'tax_tags')
```

Comment:
It would be nicer if you could skip the `filtered` when you know you don't need it :wink: